includes/class-simple-local-avatars.php

```diff
@@ -371,12 +371,15 @@ public function get_simple_local_avatar_url( $id_or_email, $size ) {
 			if ( ! $avatar_full_path ) {
 				return '';
 			}
+
```

Comment:
@jayedul  Can we make following improvements to this function:

- Can we remove `! empty( $local_avatars['media_id'] )` check on line `357` because we already validate it last line `340`?
- We should generate a `full` image URL only if:
   - The `full` image URL does not exist
   - The `full` image URL does not point to the same site upload directory